koocook_core/models/user/__init__.py

```diff
@@ -8,21 +8,26 @@
 
 
 def _default_preferences():
-    return dict()
+    return dict
 
 
 class KoocookUser(SerialisableModel, models.Model):
     exclude = ('user', 'preferences', 'user_settings')
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     # author from Author's OneToOneField
-    preferences = fields.JSONField(default=_default_preferences)
-    user_settings = fields.JSONField(default=_default_preferences)
+    preferences = fields.JSONField(default=_default_preferences())
+    user_settings = fields.JSONField(default=_default_preferences())
```

Comment:
Calling the function would result in an empty `dict` which is the same result as `default={}` which may cause problems because it is mutable.